admin/cron_lock.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import click
+
+
+CRON_LOGS_DIR = "/logs"
+CRON_LOCK_FILE = os.path.join(CRON_LOGS_DIR, "cron.lock")
+
+
+def sanity_check():
+    """ Check to make sure the logs dir exists and that the CONTAINER_NAME env
+        var is set correctly for the cron container. If the PROD_ENV var is not
+        set, assume that we are not running in production and exist with success. """
+
+    if "PROD_ENV" not in os.environ:
```

Comment:
where is this environment variable set? did you mean to check `DEPLOY_ENV=prod` ?

Reply:
I really fucking hate dyslexia. :facepalm: